2.wave3_data/3.cellprofiler_analysis/create_edit_illum_loaddata_csv.ipynb

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
Can you add a bit more documentation (like 1 sentence) here about why this is necessary? (I know you have it written somewhere else, so maybe adding here is redundant?)

Reply:
It is stated in the README, but I don't think providing as much clarity as possible hurts, so I added a sentence.